RELEASE.md

```diff
@@ -8,6 +8,16 @@
 
 In the below example, we will pretend that we're upgrading from `0.11.0` to `0.12.0`. Please use the CORRECT release numbers as these are just example numbers.
 
+## Release timeline
+
+Below is what a typical release week may look like:
+
+- **Monday (Notify):** 48-hour notification. Communicate to maintainers and public channels a release will be cut on Wednesday and to merge any pending PRs. Inform QE team. Start work on blog post as it is usually the longest part of the release process.
+- **Tuesday (Staging, Testing & Blog):** Stage the release (see instructions below) to create a new cut of the release to test. Test the pre-release (master branch) build briefly. Get feedback from QE (if applicable). Push the blog post for review (as it usually takes a few back-and-forth reviews on documentation).
```

Comment:
```suggestion
- **Tuesday (Staging, Testing & Blog):** Stage the release (see instructions below) to create a new cut of the release to test. Test the pre-release (main branch) build briefly. Get feedback from QE (if applicable). Push the blog post for review (as it usually takes a few back-and-forth reviews on documentation).
```

Reply:
`Get feedback from QE (if applicable)` --> probably say as well Others as it can be any committers